### PR TITLE
Instructor: view results: fix bug: sorting results by team is broken #8837

### LINKED ISSUE #8837: Instructor: view results: fix bug: sorting results by team is broken
v6.5.1, Windows 10, Chrome, live server

Steps:
1. Login as instructor
1. Go to the results of a session. Expand results of a question and sort individual responses by giver team

Problem: the table gets messed up, as follows.

image

Other observations: seem to affect only some type of questions. Didn't observe the problem in contribution questions.

### PR DESCRIPTION
**Pull request by tshradheya**
Fixes #8837

**Outline of Solution**

- Initially the statement `const $rowList = $('tr', $table);` found all rows of the `$table` object which included all rows of child table (Part of the `add Comment` modal for the instructor) and hence it extracted and sorted them as well.

- This also results in simpler logic to query rows which are to be sorted

- This PR queries only the rows inside required tbody which are to be sorted and hence doesn't add any more content thereby fixing the bug.

P.S. I will be investigating soon why no tests took care of this and add extra tests in future PR as this is of urgent priority and should be merged ASAP

### CODE REVIEW

**Comment by tshradheya** [comment]
@TEAMMATES/seniordevs Ready for review

**Comment by tshradheya** [comment]
Reminder to review this

File: src/main/webapp/dev/js/common/sortBy.js [hunk 1]
```diff
@@ -219,18 +219,14 @@ class TableButtonHelpers {
...
- *     Ignore rows above this when sorting. Start sorting from this row.
- *     The main use case for this is to ignore the first row, which usually contains the table headers.
- *     In that case, set rowOffset to 1 (thus ignoring row 0).
  */
-function sortTable($table, colIdx, comparatorOrNull, extractorOrNull, shouldSortAscending, rowOffset) {
+function sortTable($table, colIdx, comparatorOrNull, extractorOrNull, shouldSortAscending) {
     let columnType = 0;
     let store = [];
-    const $rowList = $('tr', $table);
+    const $rowList = $table.find('>tbody > tr');
```
Comment (LiHaoTan) [inline src/main/webapp/dev/js/common/sortBy.js]: Minor nit: $table.find('> tbody > tr');

**Comment by tshradheya** [comment]
@wkurniawan07 Can final review this? Prof @damithc is out of station

**Comment by tshradheya** [comment]
Prof @damithc, this is ready for final review

**Pull request merged**